Rog/src/main/java/springbootMigracion/java/com/example/demo/configuration/SwaggerConfig.java

```diff
@@ -0,0 +1,22 @@
+package springbootMigracion.java.com.example.demo.configuration;
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.servers.Server;
+
+@OpenAPIDefinition(
+        info = @Info(
+                title = "API INSTRUMENTOS FINANCIEROS",
+                description = "Aplicacion que permite realizar un CRUD de Instrumentos financieros e Inversores",
+                version = "1.0.0"
+        ),
+        servers = {
+                @Server(
+                        description = "DEV SERVER",
+                        url = "http://localhost:8080"
```

Comment:
No deberiamos de dejar el puerto hardocdeado en el config si lo estamos confiugrando desde el application properties. Para esto podemos leerlo utilizando la configuracion de swagger en el application properties y configurarlo dinamicamente aca:

```suggestion
url = "http://localhost:{puerto}",
            variables = {
                @ServerVariable(name = "puerto", defaultValue = "${swagger.port}")
            }
```

En el application properties agregamos esta propiedad de la siguiente forma:

swagger.port=8080